classes/output/login.php

```diff
@@ -23,23 +23,27 @@
 
 namespace local_login\output;
 
+use cache;
+use Mustache_Engine;
+use Mustache_Exception;
+
 /**
  * Output buttons.
  */
 class login {
 
     /**
-     * Output login buttons to page.
+     * Output login page.
      *
      * @param string $wantsurl
      * @return string
      */
-    public static function output_login_options($wantsurl) {
-        // Could have used a renderer, but just nicer to have everything in the same file.
+    public static function render_login($wantsurl) {
```

Comment:
I am always slightly suspicious of things called render, if we are gonna render something we may as well turn it into a proper renderer class as its pretty minnor naming tweaks